src/Sarif.Viewer.VisualStudio.Core/ErrorList/ErrorListService.cs

```diff
@@ -95,16 +95,19 @@ public static async Task ProcessLogFileAsync(string filePath, string toolFormat,
         {
             try
             {
+                Trace.WriteLine($"Processing SARIF log file `{filePath}`");
```

Comment:
>$"Processing SARIF log file `{filePath}`" [](http://example.com/codeflow?start=32&length=41)

We should put these string in Resources since we're heading for a localization requirement.

Reply:
moved the string to resources file